cocotb/share/makefiles/simulators/Makefile.verilator

```diff
@@ -28,6 +28,10 @@ ifeq ($(VERILATOR_TRACE),1)
   EXTRA_ARGS += --trace --trace-structs
 endif
 
+ifeq ($(VERILATOR_COV),1)
```

Comment:
As @themperek says, I would add this information to the documentation (see ``documentation/source/simulator_support.rst``). Many simulators have more fine-grained coverage collection options, it would be impossible to support all of them in a generic way.

Reply:
The documentation here https://cocotb.readthedocs.io/en/latest/building.html#envvar-COVERAGE made me think that at least some sort of basic coverage for verilator (and others) would be enabled when I did `COVERAGE = 1` in my makefile.  Is that still the intent?  If not (as you suggest @cmarqu this would make cocotb more complicated), then we should remove that statement from that part of the documentation too.

Reply:
Right, tracking this as part of #1495 (and resulting sub-issues).